storage_controller/src/service/safekeeper_reconciler.rs

```diff
@@ -30,31 +30,33 @@ impl SafekeeperReconcilers {
             reconcilers: HashMap::new(),
         }
     }
-    pub(crate) fn schedule_request_vec(
-        &mut self,
-        service: &Arc<Service>,
-        reqs: Vec<ScheduleRequest>,
-    ) {
+    /// Adds a safekeeper-specific reconciler.
+    /// Can be called multiple times, but it needs to be called at least once
+    /// for every new safekeeper added.
+    pub(crate) fn add_safekeeper(&mut self, node_id: NodeId, service: &Arc<Service>) {
```

Comment:
I'd name this differently: `start_reconciliation_loop` or something like that.

Reply:
will do

Reply:
Can you rename `cancel_safekeeper` as well while at it?

Reply:
renamed both to `start_reconciler` and `stop_reconciler`